linera-faucet/server/src/lib.rs

```diff
@@ -435,16 +435,26 @@ where
                         return Ok(()); // Don't return an error, so we retry.
                     }
                     chain_err => {
+                        let error_msg = format!("{}", chain_err);
+                        for request in valid_requests {
+                            let _ = request.responder.send(Err(Error::new(error_msg.clone())));
```

Comment:
(No need for `format!`?)
```suggestion
                            let _ = request.responder.send(Err(Error::new(chain_err.to_string())));
```